tests/test_hsm.py

```diff
@@ -366,6 +366,89 @@ def tearDown(self):
         del self.offset
         del self.xy0
 
+    @staticmethod
+    def computeDirectShapeFromGalSim(record, exposure, config):
+        """
+        Retrieve the shape as estimated directly by GalSim for comparison
+        purposes.
+
+        Parameters
+        ----------
+        record : `~lsst.afw.table.SourceRecord`
+            The record containing the center and footprint of the source which
+            needs measurement.
+        exposure : `~lsst.afw.image.Exposure`
+            The exposure containing the source which needs measurement.
+        config : `~lsst.meas.extensions.shapeHSM._hsm_shape.\
+                 HsmShape<Method>Config`
```

Comment:
```suggestion
                 HsmShapeConfig`
```
This will include all its subclasses. You can specify that this has to be one of the child classes in the description that follows.